sinatra/app.rb

```diff
@@ -0,0 +1,49 @@
+# sinatra setup prototype
+# OSUOSL March 2016
+
+require 'rubygems'
+require 'bundler/setup'
+require 'sinatra'
+require 'json'
+require 'date'
+
+# preprocess the json from server
+# TODO: change data source to redis
+file = File.read('sample.json')
+data = JSON.parse(file)
+
+# default helloworld
+get '/' do
+  "Hello World!"
```

Comment:
This was put in as a placeholder / cheeky joke. We might want to replace it with a *real* main page; maybe a rendered version of our README or at the very least a simple webpage that points to our project (the osuosl/iam github repo).